src/crypto/EncryptionSetup.ts

```diff
@@ -365,7 +365,7 @@ class SSSSCryptoCallbacks {
         }
         // if we don't have the key cached yet, ask
         // for it to the general crypto callbacks and cache it
-        if (this.delegateCryptoCallbacks) {
+        if (this.delegateCryptoCallbacks && this.delegateCryptoCallbacks.getSecretStorageKey) {
```

Comment:
```suggestion
        if (this?.delegateCryptoCallbacks?.getSecretStorageKey) {
```
Makes it slightly easier for the eyes. :)

Reply:
No need to check `this`.
```suggestion
        if (this.delegateCryptoCallbacks?.getSecretStorageKey) {
```

Reply:
@jaller94 Could you change it in line 381 too, so the behavior's the same in both if we don't need to check for `this`? It seems that it'll always be defined, but may be an empty object sometimes.

Reply:
If #1849 gets accepted, `delegateCryptoCallbacks` becomes optional and we should keep them like this.